_conferences/360andev-2020.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "360|AnDev"
+website: https://360andev.com/
+location: Denver, Colorado, USA
+online: true
+
+date_start: 2020-07-23
+date_end:   2020-07-24
+
+cfp_end:   2020-05-8
```

Comment:
this needs a start date. you can use today if it's already open.

also,
```suggestion
cfp_end:   2020-05-08
```

Reply:
Noted and done.

I use the date from the CFP communication I received.